### ISSUE TITLE
Option to silence react-ga console spam?

### ISSUE THREAD

**Issue by viridia** [react-ga/react-ga#87]
My goal is to get my app to produce zero console output unless something is actually wrong or I am debugging.

**Comment by kukikiloke** [comment]
@viridia I don't think the react-ga outputs the logs to console by default. Did you set the flag `debug: true` when you initialize the tracker?

**Comment by viridia** [comment]
Nope, I'm just calling:

`ReactGA.initialize('UA-83247141-01');
`

And I see this on the console:

```
[react-ga] called ga(arguments);
[react-ga] with arguments: ["require","impressionTracker"]
[react-ga] called ga(arguments);
[react-ga] with arguments: ["require","mediaQueryTracker"]
[react-ga] called ga(arguments);
[react-ga] with arguments: ["require","outboundLinkTracker"]
[react-ga] called ga(arguments);
[react-ga] with arguments: ["require","pageVisibilityTracker"]
[react-ga] called ga(arguments);
[react-ga] with arguments: ["require","urlChangeTracker"]

```

**Comment by gtamir** [comment]
@viridia The problem is that the `ReactGA.ga` method is not checking if in debug mode before logging. It requires a fix.

**Comment by yangshun** [comment]
Sorry we missed checking for `debug` flags within the `ReactGA.ga()` calls. Will fix it asap. Thanks @gtamir for identifying the problem.

**Comment by gtamir** [comment]
My pleasure @yangshun. Thanks for sharing the library!

**Comment by yangshun** [comment]
Published v2.1.2, which fixes this issue.
